hydroshare/settings.py

```diff
@@ -425,3 +425,59 @@
         'rest_framework.authentication.SessionAuthentication',
     )
 }
+
+####################
+# LOGGING SETTINGS #
+####################
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'verbose': {
+            'format' : "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s",
+            'datefmt' : "%d/%b/%Y %H:%M:%S"
+        },
+        'simple': {
+            'format': '[%(asctime)s] %(levelname)s %(message)s',
+            'datefmt' : "%d/%b/%Y %H:%M:%S"
+        },
+    },
+    'handlers': {
+        'syslog': {
+            'level': 'DEBUG',
+            'class':'logging.handlers.RotatingFileHandler',
+            'filename': '/var/log/hydroshare/system.log',
+            'formatter': 'simple',
+            'maxBytes': 1024*1024*15, # 15MB
+            'backupCount': 10,
+        },
+        'djangolog': {
+            'level': 'DEBUG',
+            'class':'logging.handlers.RotatingFileHandler',
+            'filename': '/var/log/hydroshare/django.log',
+            'formatter': 'verbose',
+            'maxBytes': 1024*1024*15, # 15MB
+            'backupCount': 10,
+        },
+        'appslog': {
+            'level': 'DEBUG',
+            'class':'logging.handlers.RotatingFileHandler',
+            'filename': '/var/log/hydroshare/apps.log',
+            'formatter': 'verbose',
+            'maxBytes': 1024*1024*15, # 15MB
+            'backupCount': 10,
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers':['syslog', 'djangolog'],
+            'propagate': True,
+            'level':'DEBUG',
+        },
+        'apps': {
+            'handlers': ['syslog', 'appslog'],
+            'level': 'DEBUG',
+        },
```

Comment:
I wasn't completely sure how to address this as a place for developers to add their own logging per app... The generic django DEBUG log is quite verbose, and could be overkill for debugging a specific portion of a specific app.

My intent was to have the `system.log` file be an account of everything, but then also allow other logs to exist as needed.

For example, there could be definitions for all of the existing apps, `hs_core`, `hs_app_netCDF`, etc., but I wasn't sure if I should try and set that up from the get go.

Perhaps a better route would be to set up an example of an additional logger as a comment, but just leave the base `system.log` and overly verbose `django.log` to start with.

Thoughts?